apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nav-sidebar/src/components/nav-sidebar/index.tsx

```diff
@@ -195,6 +206,11 @@ function WpcomBlockEditorNavSidebar() {
 			>
 				<div className="wpcom-block-editor-nav-sidebar-nav-sidebar__header">
 					<Button
+						label={ __( 'Close block editor sidebar', 'full-site-editing' ) }
```

Comment:
Here the 'label' string will be used for both the tooltip and the `aria-label` correct?

Reply:
Yeah, if you set the `label` prop then the `<Button>` makes sure that it sets the `aria-label` attribute too.